spec/controllers/idv/capture_doc_controller_spec.rb

```diff
@@ -18,6 +18,8 @@
   before do
     stub_analytics
     allow(@analytics).to receive(:track_event)
+    allow(LoginGov::Hostdata::EC2).to receive(:load).
+      and_return(OpenStruct.new(region: 'us-west-2', domain: 'example.com'))
```

Comment:
I feel like we should update the hostdata gem to make `LoginGov::Hostdata::EC2` a plain struct so we can just do like: `LoginGov::Hostdata::EC2.new(region:.., domain:...)`